src/app/questionnaire-item/questionnaire-form-item-boolean/questionnaire-form-item-boolean.component.html

```diff
@@ -8,7 +8,7 @@
     ></app-questionnaire-form-item-hint>
   </label>
   <mat-radio-group [formControl]="formControl" [required]="item.isRequired">
-    <mat-radio-button [value]="'yes'" color="primary">Yes</mat-radio-button>
-    <mat-radio-button [value]="'no'" color="primary">No</mat-radio-button>
+    <mat-radio-button [value]="true" color="primary">Yes</mat-radio-button>
```

Comment:
This could be a breaking change but as far as I tested the boolean flags never really worked (they were not even represented in the response). If it is a problem, feel free to change it back.